src/test/java/com/courtcanva/ccfranchise/services/StaffServiceTest.java

```diff
@@ -46,4 +49,16 @@ void shouldCreatedStaffSuccessful() {
         assertEquals(1232L, staffResult.getStaffId());
 
     }
+
+    @Test
+    void checkEmailIsExisted() {
```

Comment:
`shouldThrowErrorWhenEmailExits`. keep your test naming methods in a consistent way.